index.test.js

```diff
@@ -295,6 +295,33 @@ describe('SuperExpressive', () => {
     () => SuperExpressive().utf16Code('0A2K')
   );
 
+  testRegexEquality('unicodeCharCode', /\u{00021}/u, SuperExpressive().unicodeCharCode('00021'));
+  testErrorConditition(
+    'unicodeCharCode: only one digit',
+    'unicodeCharCode() can only be called with a 4 or 5 character string (got a)',
+    () => SuperExpressive().unicodeCharCode('a')
+  );
+  testErrorConditition(
+    'unicodeCharCode: invalid characters',
+    'hex can only contain hexadecimal characters (got 00A2K)',
+    () => SuperExpressive().unicodeCharCode('00A2K')
+  );
+
+  testRegexEquality('unicodeProperty', /\p{Script=Latin}/u, SuperExpressive().unicodeProperty('Script=Latin'));
+  testRegexEquality('unicodeProperty: lone property', /\p{L}/u, SuperExpressive().unicodeProperty('L'));
+  testErrorConditition(
+    'unicodeProperty: invalid characters',
+    `Property is not a valid Unicode property (got ~). ` +
+    `For valid properties see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Unicode_character_class_escape`,
+    () => SuperExpressive().unicodeProperty('~')
+  )
+  testErrorConditition(
+    'unicodeProperty: invalid property name',
+    `Property is not a valid Unicode property (got nope). ` +
+    `For valid properties see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Unicode_character_class_escape`,
+    () => SuperExpressive().unicodeProperty('nope')
+  )
+  // SuperExpressive().unicodeProperty('a').toRegex();
```

Comment:
I guess this can go

Reply:
Oh woops, a commented out line snuck through. Is that the only thing you want me to remove?

Reply:
Yeah just the comment. The other tests look good. I wasn't going to nitpick on it, but if you're removing the line, could you also add the missing semicolons in a few places :smile: